flashcards/src/styles/pages/login/login-form.css

```diff
@@ -0,0 +1,49 @@
+.main-content {
+  padding: 108px 48px 108px 93px;
+  display: flex;
+  gap: 80px;
+}
+
+.login-form {
+  min-width: 365px;
+  padding-top: 114px;
+}
+
+.login-form__body {
+  padding-top: 24px;
+}
+
+.login-form__inputs {
+  display: flex;
+  flex-direction: column;
+  margin-top: 5px;
+}
+
+.login-form__input {
+  margin-top: 11px;
+  padding: 6px 10px;
+  border-radius: 50px;
+  outline: none;
+}
+
+.login-form__input + p {
```

Comment:
So every p tags in the login-form__input will be applied this style. What happens if you have another p styles in the future?

Reply:
@hungnguyenhuu-agilityio  I have updated for this case. Please help me to check again.